packages/ui/components/editor/Editor.test.tsx

```diff
@@ -7,7 +7,7 @@ import type { TextEditorProps } from "./types";
 
 describe("Editor", () => {
   const defaultProps: TextEditorProps = {
-    getText: vi.fn(),
+    getText: vi.fn(() => ""),
```

Comment:
We should mock the correct behavior by returning a string, it was returning undefined earlier.